leo/commands/build.rs

```diff
@@ -32,10 +33,53 @@ use snarkvm_r1cs::ConstraintSystem;
 use structopt::StructOpt;
 use tracing::span::Span;
 
-/// Compile and build program command
+/// Compiler Options wrapper for Build command. Also used by other commands which
+/// require Build command output as their input.
+#[derive(StructOpt, Clone, Debug)]
+pub struct BuildOptions {
```

Comment:
This struct is duplicated [here](https://github.com/AleoHQ/leo/blob/f9e6c9d86a79715fdd508d077e8c210b95f9af8c/compiler/src/option.rs#L21)

Is the duplication necessary? If not, then you can replace instances of `CompilerOptions` -> `BuildOptions`